release/release_logs/2.2.0/benchmarks/many_actors.json

```diff
@@ -1,15 +1,32 @@
 {
-    "_peak_memory": 3.18,
-    "_peak_process_memory": "PID\tMEM\tCOMMAND\n291\t1.44GiB\t/home/ray/anaconda3/lib/python3.7/site-packages/ray/core/src/ray/gcs/gcs_server --log_dir=/tmp/ray/s\n2023\t0.85GiB\tpython distributed/test_many_actors.py\n415\t0.35GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/dashboa\n47\t0.09GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/anyscale session web_terminal_server --deploy\n717\t0.06GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/dashboard/agen\n44\t0.06GiB\t/home/ray/anaconda3/bin/python /home/ray/anaconda3/bin/jupyter-notebook --NotebookApp.token=aph0_Ckc\n2160\t0.06GiB\tray::MemoryMonitorActor.run\n342\t0.05GiB\t/home/ray/anaconda3/bin/python -m ray.util.client.server --address=172.31.240.63:9031 --host=0.0.0.0\n615\t0.05GiB\t/home/ray/anaconda3/bin/python -u /home/ray/anaconda3/lib/python3.7/site-packages/ray/_private/log_m\n542\t0.02GiB\t/home/ray/anaconda3/lib/python3.7/site-packages/ray/core/src/ray/raylet/raylet --raylet_socket_name=",
-    "actors_per_second": 600.3905147006428,
+    "_dashboard_memory_usage_mb": 427.753472,
+    "_dashboard_test_success": true,
+    "_peak_memory": 6.35,
```

Comment:
created https://github.com/ray-project/ray/issues/32090 for peak memory increase